test_scripts/API/ServiceStatusUpdateToHMI/020_Video_service_protected_PTU_FAILED_TimeOut_2_failed_external.lua

```diff
@@ -53,6 +53,10 @@ function common.onServiceUpdateFunc(pServiceTypeValue)
     end)
   :Times(2)
   :Timeout(timeout)
+
+  common.getMobileSession():ExpectNotification("OnHMIStatus",
+    { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+  :Timeout(timeout)
```

Comment:
@GetmanetsIrina Please notice script invokes `startServiceWithOnServiceUpdate_PTU_FAILED()` function 2 times.
App will be moved to NONE during 1st execution and stayed in NONE during 2nd one.
Propose to activate app before 2nd execution.
This also applicable for `021` and `022` scripts

Reply:
@dboltovskyi, please find the update in 962e6c2